### PR TITLE
Upload workflow artifacts to prerelease

### PR DESCRIPTION
**Pull request by Vinfall**
Upload workflow artifacts to a prerelease tagged "nightly", so users can download nightly builds without login. I did not touch REAMDE as you may still find nightly.link useful somehow.

It's also possible to use commit hash or whatever as tag name, simply change `tag: "nightly"` in this part. I set it to nightly as I prefer a permalink like https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/releases/download/nightly/LibreHardwareMonitor-net472.zip:
```yaml
      - name: Upload artifact to prerelease
        uses: ncipollo/release-action@v1
        with:
          # Only update attachments on "nightly" tag
          prerelease: True
          tag: "nightly"
          updateOnlyUnreleased: True
```

I tested net472 & net60 locally and they worked fine on my machine.

### CODE REVIEW

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -37,39 +38,64 @@ jobs:
...
+          # Remove old artifacts and replace with new ones
+          removeArtifacts: True
+          replacesArtifacts: True
+          # Update preferences
+          allowUpdates: True
+          omitBody: True
+          omitBodyDuringUpdate: True
+          omitNameDuringUpdate: True
+          artifacts: "outputs/LibreHardwareMonitor*.zip, bin/Release/LibreHardwareMonitorLib.*.nupkg"
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (PhyxionNL) [inline .github/workflows/master.yml:101]: Can you put this in a separate PR?

Comment (Vinfall) [reply .github/workflows/master.yml:101]: Done in #1279.

**Comment by Vinfall** [comment]
Hmm, in case I did not make it clear. #1279 only updates action versions and what you suggested is actually in _this_ PR.

**Comment by Vinfall** [comment]
Closed due to inactivity, feel free to reach out if you ever decide to ship nightly via prerelease.
